### ISSUE TITLE
Some lines are incorrectly coloured

### ISSUE THREAD

**Issue by IntelOrca** [spgennard/vscode_cobol#13]
Here are a few issues I found with colourisation of managed COBOL.

```cobol
      * enums do not get coloured correctly
       display type CobolToken::TK__CH_FULL_STOP
       display type TokenStreamBehavior::SkipCommas
      * null is not coloured
       display set return-value to (tok <> null)
      * Reserved words get coloured when they are actually type members properties
       display _startPos::Line = _endPos::Line
      * Reserved words get coloured when they are escaped with #
       display #line
      * Reserved words get coloured after underscore (underscore wrongly treated as name seperator)
      * `list` and `string` are not coloured here:
       01 _lines list[string].
      * Stack is coloured in first instance but not second instance
       01 _indentStack type Stack[binary-long] value new Stack[binary-long]().
      * Full stop after the 0 is coloured
       method-id ResetIndent.
       procedure division using by value n as binary-long = 0.
           goback
       end method.
      * `iterator-id` and `end iterator` not reserved words
       iterator-id
       end iterator
```

**Comment by spgennard** [comment]
Fixed in 3.1.0

**Comment by IntelOrca** [comment]
Just pointing out that the following isn't fixed:
```
      * Full stop after the 0 is coloured
       method-id ResetIndent.
       procedure division using by value n as binary-long = 0.
           goback
       end method.
```
